lib/hanami/extensions/view/slice_configured_view.rb

```diff
@@ -32,12 +32,52 @@ def inspect
         # rubocop:disable Metrics/AbcSize
         def configure_view(view_class)
           view_class.settings.each do |setting|
-            if slice.config.views.respond_to?(:"#{setting}")
-              view_class.config.public_send(
-                :"#{setting}=",
-                slice.config.views.public_send(:"#{setting}")
-              )
-            end
+            next unless slice.config.views.respond_to?(:"#{setting}")
```

Comment:
@timriley if you put a `!slice.parent` guard here too, the logic below could be simpler

Reply:
@solnic Thanks for the suggestion :)

I've just taken a look at this now... and I'm not sure I could move the parent check up here. We still _do_ want to apply config for slices with or without parents; we just don't want to _reapply_ the config if the slice has a parent _and_ its values match everything the parent would have already applied.

I'll leave things as they are for now, but you'd be very welcome to take a pass at this as a follow up :)

(I'll readily admit that all the work we're having to do here is more than I would like. It'd be good if we could provide some dry-configurable API to help with this kind of case in the future, i.e. knowing whether a config has been assigned or mutated)